bpf/process/types/basic.h

```diff
@@ -770,6 +774,12 @@ filter_inet(struct selector_arg_filter *filter, char *args)
 	case op_filter_notsport:
 	case op_filter_notdport:
 		return filter_32ty_map(filter, (char *)&port);
+	case op_filter_sportpriv:
+	case op_filter_dportpriv:
+		return port < 1024;
```

Comment:
Would be nice to have an LT,GT,LEQ,GEQ operators for ports. For one, we would not need to hardcode the privileged port. Also, it would make it easier to handle port ranges without inserting all the values in the map. Just a thought, not a blocker -- we can revisit this if needed.

Reply:
I couldn't think of a use case for LT,GT,LEQ,GEQ on their own other than privileged ports. Typically port ranges are more useful.  For ex, >60000 is not going to be used much, but 60000-60999 might be.
I guess my concern with combining LT,GT,LEQ,GEQ to create ranges is where you want to specify two or more ranges of ports: e.g. 0-200, 30000-30099, 50100-50110. This either needs three selectors or needs a loop to walk three entries in a single selector. I think this is better served by a port lookup map, even if that map gets large – it's just memory, and each encoded port is notionally four bytes. I'd rather reduce CPU hit and reduce the number of selectors at the expense of memory I think, given we're talking small amounts and not megabytes.

Reply:
FYI, @aspsk has a nice proposal about doing this: https://lore.kernel.org/bpf/20220907080140.290413-1-aspsk@isovalent.com/